sqlair.go

```diff
@@ -35,9 +35,16 @@ type S []any
 var ErrNoRows = sql.ErrNoRows
 var ErrTXDone = sql.ErrTxDone
 
+// stmtCache stores the driver prepared statements associated to the SQLair
```

Comment:
I would change all these "driver prepared statements" to "driver-prepared statements".